CMakeLists.txt

```diff
@@ -354,6 +354,12 @@ if(WITH_BLKIN)
   include_directories(SYSTEM src/blkin/blkin-lib)
 endif(WITH_BLKIN)
 
+option(WITH_JAEGER "Use jaeger to create tracepoints and display it on Jaeger
+UI" ON)
```

Comment:
how is this option connected with "display it on Jaeger UI"? do we have a builtin Jaeger UI support?

Reply:
jaeger provides a docker image with agent, collector and frontend. So, in this case Ceph will generate the spans, while an external agent will work on sending them to a collector. which will then render the span to Jaeger UI.

Reply:
Jaeger UI is not part of what is implemented using WITH_JAEGER. it can be a part of the description of the docker image you are referencing though.